config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-release-1-5.yaml

```diff
@@ -291,12 +313,11 @@ presubmits:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-1.5
       testgrid-tab-name: capi-pr-e2e-full-release-1-5
   - name: pull-cluster-api-e2e-workload-upgrade-1-27-1-28-release-1-5
+    cluster: eks-prow-build-cluster
     labels:
       preset-dind-enabled: "true"
       preset-kind-volume-mounts: "true"
     decorate: true
-    decoration_config:
-      gcs_credentials_secret: "" # Use workload identity for uploading artifacts
     always_run: false
     branches:
     # The script this job runs is not in all branches.
```

Comment:
I think we're missing something in l.309++

Reply:
Same in L.353++ and l.389++

Reply:
@sbueringer I force pushed little early. I'll add additional commits and will squash later. 

I have used 32Gi for all e2e jobs in the latest commit. 
I've made sure that there's no usage of `decorate_config` or `gcs_credentials_secret` in the jobs.